### ISSUE TITLE
NGINX listens on port which is not exposed

### ISSUE THREAD

**Issue by DiCanio** [num-codex/codex-deploy#8]
In 92fa7978 a change to the reverse proxy was introduced. It changed the proxy's exposed port from `8443` to `8080`. However no configuration changes have been made.

At this point the current solution should not work since NGINX is listening on a port that is not exposed by the container. The port NGINX is listening on should be aligned with any exposed port.

**Comment by DiCanio** [comment]
For further clarification keep reading:

Excerpt of NGINX config:
```
server {
  listen       8443 ssl;
```

Excerpt NGINX docker-compose:
```
ports:
  - ${PORT_NUM_NODE_REV_PROXY:-444}:8080
```

**Comment by DiCanio** [comment]
Fixed in #9.
